bin/get_data.js

```diff
@@ -0,0 +1,60 @@
+const fs = require('fs')
+const process = require('process')
+const TOML = require('@iarna/toml')
+
+const header = 'function addEventListener() {}; ';
+const footer = ' exports.experimentPages = experimentPages; ';
+const body = fs.readFileSync('./workers/redirector.js');
+var experimentPages = eval(header + body + footer);
+
+const stage_domain = "https://www.allizom.org"
```

Comment:
Nit: try and use camelCase and single quotes for consistency. Always use semi-colons (we should probably define some rules in our `.eslintrc.js` config here).

```suggestion
const stageDomain = 'https://www.allizom.org';
```